### PR TITLE
Feat - product list page

### PR DESCRIPTION
**Pull request by biswanath-tewari**
fix - custom slider width 
features:-
1. loader component
2. filter
3. product list page

### CODE REVIEW

File: src/backend/db/products.js [hunk 1]
```diff
@@ -33,8 +41,11 @@ export const products = [
     price: '80,000',
```
Comment (sharathnair9999) [inline src/backend/db/products.js:41]: instead of two separate variables you can put only one integer and use .toLocaleString() to get the commas in UI

**Review by sharathnair9999 — APPROVED** [review]
Code looks clean. Gave a small sugeestion. Also try to check why the deploy is getting failed

File: src/backend/db/products.js (hunk 1 above)
Comment (biswanath-tewari) [reply src/backend/db/products.js:41]: Thanks for the suggestion, I have refactored it. And yes I have an idea why the deployment is failing

**Review by SukanyaSen530 — APPROVED** [review]

**Review by sandeepkumardev-zz — APPROVED** [review]
good code format with scss

**Pull request merged**